tools/firely-auth/firely-auth-mfa.rst

```diff
@@ -0,0 +1,24 @@
+.. _firely_auth_mfa:
+
+Using Multi Factor Authentication in Firely Auth
```

Comment:
We should mention that in our case 2FA means using a time-based one-time password